contrib/review-trailers-tooling/README.md

```diff
@@ -0,0 +1,41 @@
+## What's this?
+
+A basic set of tools to ease adding review trailers, in order to acknowledge reviewers' work.
```

Comment:
Maybe add a link to something like https://git.wiki.kernel.org/index.php/CommitMessageConventions? I don't think trailers are too common, but that might just be my experience.